Client/src/app/components/historyComponent/history.component.ts

```diff
@@ -0,0 +1,34 @@
+import {Component, EventEmitter, Output} from '@angular/core';
+import {HttpService} from '../http.service';
+import {PlayerService} from '../player.service';
+
+@Component({
+  selector: 'history-component',
+  templateUrl: './history.component.html',
+  styleUrls: ['./history.component.less']
+})
+
+export class HistoryComponent {
+  historyReady = false;
+
+  constructor(private httpService: HttpService,
+              private playerService: PlayerService) { }
+
+  async ngOnInit() {
+    await this.getAllHistory();
+  }
+
+  async getAllHistory() {
```

Comment:
a little bit peaky, but I would have called this getVideosHistory and deleteVideoFromHistory instead of deleteHistory,
Also the component should be called VideosHistoryComponent (or some stuff in this area).
The reason is that the term 'History' now does not have a clear meaning - is it the list of videos from the user's history? a single entry from the history? what kind of history?